lib/thrift/binary/framed/server.ex

```diff
@@ -8,10 +14,45 @@ defmodule Thrift.Binary.Framed.Server do
           | {:max_seconds, non_neg_integer}
           | {:tcp_opts, :ranch_tcp.opts()}
           | {:ssl_opts, [Thrift.Transport.SSL.option()]}
+          | {:transport_opts, :ranch.opts()}
 
   @type server_opts :: [server_option]
 
-  @spec start_link(module, 1..65_535, module, [server_option]) :: GenServer.on_start()
+  @doc """
+  Starts the server using the specified handler module.
+
+  The following server options can be specified:
+
+  `:worker_count`: The number of acceptor workers used to take requests.
+
+  `:name`: (Optional) The name of the server. The server's pid becomes
+  registered under this name. If not specified, the handler module's name
+  is used.
+
+  `:max_restarts`: The number of times to restart (see `:max_seconds`).
```

Comment:
I don't think we need to expose these. Instead set `max_restarts: 0` and allow the parent supervisor to handle things. The supervisor this configures is hidden from the user.

Reply:
That sounds like more than a documentation, yes? I'll remove these from the documentation here, and we can do the other bits separately.

Reply:
Done in #400.